### PR TITLE
Migrate storage/aoco_alter/aoco_alter_sql_test tests to ICG.

### PR DESCRIPTION
**Pull request by slari-pivotal**
There is considerable overlap between the aoco_alter/aoco_alter_sql_test tests
and the test/regress/*/uao_dll tests. This is the first commit to move several
of the aoco_alter/aoco_alter_sql_test tests to ICG.

This commit moves alter_ao_table_oid_column.sql and alter_ao_table_oid_row.sql
from sql/uao_ddl directories to sql/uao_col and sql/uao_row directories
respectively.

After this change, the sql/uao_ddl and expected/uao_ddl directories are only
used for generation of intermediate .sql and .out files during the processing of
the --ao-dir option of regress.  Therefore, I added .gitignore files in these
directories.

### CODE REVIEW

File: src/test/regress/sql/uao_col/bulk_dense_content_rle_compress.sql [hunk 1]
```diff
@@ -0,0 +1,11 @@
+-- Bulk dense content header with RLE compression
```
Comment (ashwinstar) [inline src/test/regress/sql/uao_col/bulk_dense_content_rle_compress.sql]: We can add these SQLs to rle.sql file

File: src/test/regress/sql/uao_col/small_content_rle_compress_hdr.sql [hunk 2]
```diff
@@ -0,0 +1,11 @@
+-- Small content header with RLE header inside
```
Comment (ashwinstar) [inline src/test/regress/sql/uao_col/small_content_rle_compress_hdr.sql]: We can add these SQLs to rle.sql file

File: src/test/regress/sql/uao_ddl/README [hunk 3]
```diff
@@ -0,0 +1,7 @@
+This directory only contains generated files created by running the
+
+	 ./pg_regress --ao-dir=uao ...
```
Comment (ashwinstar) [inline src/test/regress/sql/uao_ddl/README]: ./pg_regress --ao-dir=uao_ddl ...

**Comment by slari-pivotal** [comment]
This has been merged to master.

**Pull request closed without merging**